buildscripts/rethinkdbintegrationtest.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
Wondering if we should just modify `integrationtest.sh` to take a compose file argument?  This seems otherwise the same except for that parameter.